### PR TITLE
Fixes from feedback document 

### PR DESCRIPTION
**Pull request by subinasr**
## Changes

* Fix minor stylings
* Add questionnaire metadata display

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

### CODE REVIEW

File: src/components/ChoiceCollectionSelectInput/index.tsx [hunk 1]
```diff
@@ -132,13 +137,30 @@ function ChoiceCollectionSelectInput<
...
+    const handleChoiceAddSuccess = useCallback((newChoice: ChoiceCollectionType) => {
+        otherProps?.onChange(newChoice.id, name);
+
+        if (otherProps?.onOptionsChange) {
+            otherProps?.onOptionsChange((oldOptions) => (
+                [
+                    ...(oldOptions ?? []),
+                    newChoice,
+                ]
+            ));
```
Comment (AdityaKhatri) [inline src/components/ChoiceCollectionSelectInput/index.tsx]: When newChoice is updated (not added), chances are the oldOptions might already have that option. Make sure you search and remove that before you add the newChoice(updated choice) in the list.

File: src/views/Home/QuestionnaireItem/index.tsx [hunk 2]
```diff
@@ -241,6 +261,31 @@ function QuestionnaireItem(props: Props) {
...
+
+    const {
+        data: questionsCountResponse,
+    } = useQuery<QuestionsCountQuery, QuestionsCountQueryVariables>(
+        QUESTIONS_COUNT,
+        {
+            skip: isNotDefined(questionsCountVariables),
+            variables: questionsCountVariables,
+        },
+    );
```
Comment (AdityaKhatri) [inline src/views/Home/QuestionnaireItem/index.tsx]: Add fixme to fetch this from the server

File: src/views/QuestionnaireEdit/IntegerQuestionForm/index.tsx [hunk 3]
```diff
@@ -130,7 +129,9 @@ const schema: FormSchema = {
             required: true,
             requiredValidation: requiredStringCondition,
         },
-        hint: {},
+        hint: {
+            defaultValue: '',
+        },
```
Comment (AdityaKhatri) [inline src/views/QuestionnaireEdit/IntegerQuestionForm/index.tsx:134]: This is not required as this is probably set in the form schema as well.

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

File: src/views/About/index.tsx [hunk 4]
```diff
@@ -313,16 +282,25 @@ export function Component() {
...
-                                leafNodes={dimensions.filter(
-                                    (dimension) => dimension.category1 === rowItem[0].category1,
-                                )}
-                                subDimensions={rowItem}
-                                sectors={uniqueSectorList}
-                                setRightPaneShown={setRightPaneShown}
-                                setSelectedLeafGroupIds={setSelectedLeafGroupIds}
-                            />
+                        {dimensionRows.map((subDimensionsWithinRow) => (
+                            <>
```
Comment (AdityaKhatri) [inline src/views/About/index.tsx]: Add unique key in each mapped item

File: src/views/Home/QuestionnaireItem/index.tsx [hunk 5]
```diff
@@ -404,7 +398,82 @@ function QuestionnaireItem(props: Props) {
...
+                <div className={styles.metadata}>
+                    {(questionnaireItem.dataCollectionMethodsDisplay.length > 0) && (
+
+                        <TextOutput
+                            className={styles.metadatum}
+                            label="Data Collection Technique"
+                            valueContainerClassName={styles.tags}
+                            value={questionnaireItem.dataCollectionMethodsDisplay.map(
+                                (method) => (
+                                    <span className={styles.tag}>
```
Comment (AdityaKhatri) [inline src/views/Home/QuestionnaireItem/index.tsx]: Chabi kidhar hai, chabi?

File: src/views/Home/QuestionnaireItem/index.tsx [hunk 6]
```diff
@@ -404,7 +398,82 @@ function QuestionnaireItem(props: Props) {
...
+                            hideLabelColon
+                            block
+                        />
+                    )}
+                    {isDefined(questionnaireItem.requiredDuration) && (
+                        <TextOutput
+                            className={styles.metadatum}
+                            label="Estimated time"
+                            value={(
+                                <span className={styles.tag}>
```
Comment (AdityaKhatri) [inline src/views/Home/QuestionnaireItem/index.tsx:452]: Chabi kidhar hai, chabi?

File: src/views/Home/QuestionnaireItem/index.tsx [hunk 7]
```diff
@@ -404,7 +398,82 @@ function QuestionnaireItem(props: Props) {
...
+                            hideLabelColon
+                            block
+                        />
+                    )}
+                    {(questionnaireItem.enumeratorSkillsDisplay.length > 0) && (
+                        <TextOutput
+                            className={styles.metadatum}
+                            label="Enumerator Skill"
+                            valueContainerClassName={styles.tags}
+                            value={questionnaireItem.enumeratorSkillsDisplay.map(
```
Comment (AdityaKhatri) [inline src/views/Home/QuestionnaireItem/index.tsx:467]: Chabi kidhar hai, chabi?

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

File: src/views/Home/QuestionnaireItem/index.tsx (hunk 6 above)
Comment (subinasr) [reply src/views/Home/QuestionnaireItem/index.tsx:452]: Idhar lock ich nahi hai, chabi se kyaa karoge?

**Pull request merged**